src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionControllerGetYourTranscriptsLegacyIntTest.java

```diff
@@ -54,10 +70,148 @@ void beforeEach() {
 
         testUser = dartsDatabase.getUserAccountStub().getIntegrationTestUserAccountEntity();
         systemUser = dartsDatabase.getUserAccountStub().getSystemUserAccountEntity();
+
+    }
+
+    @Test
+    void getYourTranscripts_ShouldReturnRequesterOnly_WithTranscriptionUrgency() throws Exception {
```

Comment:
Maybe improve the test name to be clearer about what it's doing?

```suggestion
    void getYourTranscripts_shouldReturnRequesterTranscriptions_whenNoLinkedHearing() throws Exception {
```

Reply:
done